app/mailers/user_emailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserEmailer < ApplicationMailer
+  default from: 'laurawhalin@gmail.com'
```

Comment:
I took the one out in application_mailer. I think it would be good to have as a backup in case your app heavily relies on emailing? But that's a guess.